### DISCUSSION TITLE
Cycling tabs in recently used order

### DISCUSSION THREAD

**Discussion by aksh1618** [kovidgoyal/kitty#3720] [Q&A] [answered]
Hi! I'm exploring and setting up kitty to match my workflow. The features (speed, configuration, kittens, so many awesome things!), ideology, community and how active @kovidgoyal is, made me want to switch to Kitty (along with a broken progress bar rendering in my existing terminal, ofc), and I'm loving it so far! I just have a couple things left to figure out, that I'm used to in my current workflow, one of them is being able to cycle between two recently used tabs, instead of having to go through the whole stack. Is there a way to do this in kitty?

**Comment by kovidgoyal** [comment, marked as answer]
map f1 goto_tab -1

will take you to the previously active tab.

**Reply by aksh1618** [reply]
Thanks, exactly what I needed! I was sure kitty would support it :heart:
